i18n/en.yaml

```diff
@@ -44,6 +44,6 @@
   translation: Enter Your Email To Subscribe
 - id: newsLetterEngagment
   translation: I want to received the monthly newsletter and other updates from Rotational. You
```

Comment:
```suggestion
  translation: I want to receive the monthly newsletter and other updates from Rotational Labs. You
```

Reply:
A minor grammar update. Using Rotational Labs instead of Rotational may be more consistent.